lib/services/role.go

```diff
@@ -2515,6 +2546,51 @@ func (set RoleSet) CheckAccessToRule(ctx RuleContext, namespace string, resource
 	})
 }
 
+// GetKubeResources returns allowed and denied list of Kubernetes Resources configured in the RoleSet.
+func (set RoleSet) GetKubeResources(cluster types.KubeCluster) (allowed, denied []types.KubernetesResource) {
+	var (
+		allowedResources []types.KubernetesResource
+		deniedResources  []types.KubernetesResource
+	)
+	getRoleLabels := types.Role.GetKubernetesLabels
```

Comment:
Why not do `role.GetKubernetesLabels(types.Allow)` or `role.GetKubernetesLabels(types.Deny)`, as needed?